components/Tabs/src/private/MDCItemBar.m

```diff
@@ -282,13 +281,15 @@ - (void)layoutSubviews {
   const CGRect bounds = self.bounds;
 
   _collectionView.frame = bounds;
+  [_collectionView sendSubviewToBack:_selectionIndicator];
```

Comment:
I typically would expect to only see sizing / frame changes in layoutSubviews.
Please add a comment on why we reordering our subview's hierarchy.